### DISCUSSION TITLE
post/skeleton-ui-idea/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#12] [Announcements]
# post/skeleton-ui-idea/

카카오페이에서 프론트엔드 개발을 하며 스켈레톤 UI와 사용자 경험 향상에 대해 고민한 내용을 공유합니다.

https://tech.kakaopay.com/post/skeleton-ui-idea/

**Comment by kimpra2989** [comment]
지연이 있는 컴포넌트에 대해선 그에 대한 처리를 무조건적으로 해야한다고만 생각을 해왔는데, 처리를 안 하는 것이 오히려 사용성을 개선하게 될 수도 있다는 점이 흥미롭네요.
